protostar/commands/test/test_command.py

```diff
@@ -120,11 +122,26 @@ def arguments(self) -> List[Command.Argument]:
                 description="Print slowest tests at the end.",
                 default=0,
             ),
+            Command.Argument(
+                name="last-failed",
+                short_name="lf",
+                type="bool",
+                description="only re-run the failures.",
```

Comment:
```suggestion
                description="Only re-run failed and broken test cases.",
```

I think broken test cases could be also re-run.

Reply:
[update description](https://github.com/software-mansion/protostar/pull/895/commits/50d524648d0da7c9f7c93aee761fa438878bd354)